js/components/controlBar.js

```diff
@@ -219,6 +221,56 @@ var ControlBar = React.createClass({
     this.props.controller.state.focusedControl = null;
   },
 
+  /**
+   * Will handle the keydown event when the controlBar is active and it will restrict
```

Comment:
Can you add in that this only handles the edge case for wrapping at the beginning or the end.  The OS handles the rest.